esp/bindings/http/platform/httpservice.cpp

```diff
@@ -2389,7 +2391,7 @@ EspAuthState CEspHttpServer::authExistingSession(EspAuthRequest& authReq, unsign
     {
         time_t timeoutAt = createTime + authReq.authBinding->getServerSessionTimeoutSeconds();
         sessionTree->setPropInt64(PropSessionTimeoutAt, timeoutAt);
-        ESPLOG(LogMin, "Updated %s for (/%s/%s) : %" PRId64 "", PropSessionTimeoutAt, authReq.serviceName.isEmpty() ? "" : authReq.serviceName.str(),
```

Comment:
I'm conflicted about this one.
It feels like we might be masking an important event. If the comment on L2380 is correct, we're trying to report an interactive request to update the session timeout.
@GordonSmith is it possible these requests are coming from eclwatch withoug the "ECLWatchAutoRefresh" annotation?

Reply:
I read this to mean the `PropSessionTimeoutAt` is being set forward in the case when this is *not* an ECLWatchAutoRefresh request. I don't see any other special handling of the ECLWatchAutoRefresh property, so perhaps those explicit auto refresh requests are being handled in a different code path where they're identified another way.

Reply:
It seems the original developer thought it was an important event worth logging every time.

if this condition is very common place and not worth alerting, then I agree with the change:
```
(!sessionTree->getPropBool(PropSessionTimeoutByAdmin, false)
&& (autoRefresh.isEmpty() || strieq(autoRefresh.str(), "0")))
 ```
However, if that condition is not meant to be very common, then I wondered if it's being triggered by eclwatch, but not properly annotated with the 'ECLWatchAutoRefresh'

A quick search of the repo only found that term in the httpservice file:
![image](https://github.com/user-attachments/assets/463ffc1e-05a7-48c9-bb9c-0d8c9668cb56)
I think it's worth getting confirmation from the UI team if this param is actively being used.

Reply:
@jeclrsg I don't see `ECLWatchAutoRefresh` appearing in the Visualization codebase either. Do you know if it is used in UI at all?